docs/guide-end-of-form.rst

```diff
@@ -114,38 +115,55 @@ You can also use :ref:`draft names <instance-name>` to include information about
 
 If you use a constraint as in the above example, the user will need to come back into the draft and change their answer to the question before they can send the data.
 
-If your users have many drafts and will only need to edit a few before sending them, you can use a ``note`` without a constraint to guide users to save as draft. They will then be able to use the :ref:`bulk finalization <bulk-finalizing-drafts>` functionality when they are ready to submit.
+If your users have many drafts, will only need to edit a few before sending them, but don't know which ones will need to be edited, you can use a ``note`` without a constraint to guide users to save as draft.
```

Comment:
You are missing `and` after the first comma.

Reply:
It's a list with three items. Maybe changing the but to and is clearer?